database/database.go

```diff
@@ -285,9 +285,9 @@ func (query *SwapQuery) ToWhereClauseWithExisting(conditions []string, values []
 		}
 		conditions = append(conditions, "state IN ("+strings.Join(states, ",")+")")
 	}
-	if query.IsAuto != nil {
+	if query.Include != boltzrpc.IncludeSwaps_ALL {
```

Comment:
Where is the clause for manual only?

Reply:
if its not all then we query by `isAuto == query.Include == boltzrpc.IncludeSwaps_AUTO` - so we cover both in that branch